portfolio/src/main/webapp/index.html

```diff
@@ -98,61 +101,61 @@ <h2>Travel Log</h2>
                 <!-- Full-width images with number and caption text -->
                 <div id="first-slide" class="fade">
                     <div class="numbertext">1 / 10</div>
-                    <img src="/images/Shenzhen.jpg" style="width:100%">
+                    <img src="./images/Shenzhen.jpg" style="width:100%">
```

Comment:
Since all images have style width:100%, you may consider instead of adding it every time, add a single CSS rule like
.fade img {
    width: 100%;
}